site/src/components/progress-bar.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Progress Bar
+description: The progress-bar component in the Atlas Design System
+template: standard
+figmaEmbed:
+classType: Component
+classPrefixes:
+  - progress-bar
+---
+
+# Progress Bar
+
+Use the progress-bar component to communicate how much of a task has been completed.
+
+## Usage
+
+Here is an example of a standard usage of the progress bar component.
+
+```html
+<div class="progress-bar-container">
```

Comment:
Let's go with [`<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) usage in markup

Reply:
I don't think I can do this because <progress> is built in already https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

If I try that change, the atlas page now looks like 
![image](https://github.com/user-attachments/assets/e3898b84-8b58-4a9d-965e-1c9afe4cd081)

Reply:
Sorry, I don't understand what's the issue here. Can you elaborate?

Reply:
I can't use `<progress class="progress-bar-container">`, is that what you were asking?

Reply:
I meant to use progress element and standard markup and build styling around it. It will require revisiting the structure you made so far.

```
<label for="progress-bar">Label:</label>
<progress id="progress-bar" max="100" value="70">70%</progress>
```

Looks much cleaner and don't require inline styling and additional attributes to make it work:

```
<label for="progress-bar-demo">Label:</label>
<div id="progress-bar-demo">
	<div role="progressbar" style="width: 75%;" aria-label="demo progress bar" aria-valuenow="75"></div>
</div>
```